src/status_im2/contexts/wallet/common/utils.cljs

```diff
@@ -4,3 +4,25 @@
 (defn get-first-name
   [full-name]
   (first (string/split full-name #" ")))
+
+(defn calculate-raw-balance
+  [raw-balance decimals]
+  (if (not (js/isNaN (js/parseInt raw-balance)))
+    (/ (js/parseInt raw-balance) (Math/pow 10 (js/parseInt decimals))) 0))
+
+(defn calculate-balance
```

Comment:
is it really util, or it will be used only once in the subscription? if so better to move it there

Reply:
Done